### PR TITLE
support live withdraw fee

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: reservesetting/storage/postgres/sql.go [hunk 1]
```diff
@@ -585,14 +585,18 @@ func assetExchangeStatements(db *sqlx.DB) (*sqlx.NamedStmt, *sqlx.NamedStmt, *sq
...
 
-	const getAssetExchangeBySymbolQuery = `SELECT
-		asset_exchanges.asset_id as id,
-		asset_exchanges.symbol as symbol,
-		a.decimals as decimals	
-	FROM asset_exchanges
-		LEFT JOIN assets a on asset_exchanges.asset_id = a.id
-	WHERE asset_exchanges.exchange_id = $1
-	AND asset_exchanges.symbol= $2`
+	const getAssetExchangeBySymbolQuery = `SELECT id,
```
[the remark this answers is not in the document]
Comment (NNagato) [reply reservesetting/storage/postgres/sql.go:588]: we didn't use it before, so don't worry about it

File: reservesetting/cmd/main.go [hunk 2]
```diff
@@ -117,19 +161,48 @@ func run(c *cli.Context) error {
...
+	if err != nil {
+		sugar.Errorw("cannot get assets", "err", err)
+		return
+	}
+	for _, asset := range assets {
+		for _, ae := range asset.Exchanges {
+			if le, ok := exchanges[v1common.ExchangeID(ae.ExchangeID)]; ok {
+				withdrawFee, err := le.GetLiveWithdrawFee(asset.Symbol)
+				if err != nil {
+					sugar.Errorw("cannot get live withdraw fee", "err", err, "asset", asset.Symbol)
```
Comment (secmask) [inline reservesetting/cmd/main.go:197]: should also include assetExchange symbol in log.

File: reservesetting/http/setting_change.go [hunk 3]
```diff
@@ -59,11 +59,18 @@ func (s *Server) fillLiveInfoSettingChange(settingChange *common.SettingChange)
 		switch o.Type {
 		case common.ChangeTypeCreateAsset:
 			asset := o.Data.(*common.CreateAssetEntry)
-			for _, assetExchange := range asset.Exchanges {
+			for index, assetExchange := range asset.Exchanges {
 				err = s.fillLiveInfoAssetExchange(assets, assetExchange.ExchangeID, assetExchange.TradingPairs, assetExchange.Symbol, assetExchange.AssetID)
 				if err != nil {
 					return fmt.Errorf("position %d, error: %v", i, err)
 				}
+				if assetExchange.WithdrawFee == 0 {
```
Comment (secmask) [inline reservesetting/http/setting_change.go:67]: we will always fill withdraw fee from exchange, not manual anymore, so we dont have to check it here, considerto remove it from input too.

**Comment by NNagato** [comment]
@secmask I made the change in https://github.com/KyberNetwork/reserve-data/pull/1054/commits/8131c281899ec009438fc3bbc106fd23bc7d437c, I removed withdraw fee in update_input but still keep it in create_input (it will be overwritten with live information) because currently, we're storing the raw input directly to DB

**Pull request merged**